src/main/java/com/tagtraum/perf/gcviewer/imp/DataReaderSun1_6_0.java

```diff
@@ -620,6 +620,8 @@ protected AbstractGCEvent<?> parseLine(String line, ParseInformation pos) throws
             // pre-used->post-used, total, time
             ZonedDateTime datestamp = parseDatestamp(line, pos);
             double timestamp = getTimestamp(line, pos, datestamp);
+            // Support for PrintGCID
+            parseId(line, pos);
```

Comment:
please rename the method (e.g. parseGcId()) and remove the comment.

Reply:
Done